cpu/esp32/Kconfig.esp32

```diff
@@ -24,7 +24,7 @@ config CPU_FAM_ESP32
 
     select PACKAGE_ESP32_SDK if TEST_KCONFIG
 
-    select MODULE_LIBC_GETTIMEOFDAY if TEST_KCONFIG
```

Comment:
This line was already removed in PR #18202 with 4dc5e86b2a5e986e206416f9b1e7c26bfb55d882 but readded by mistake when `Kconfig` was splitted into `Kconfig` and `Kconfig.esp32` with PR #18320 as a split-off from PR #17842 :worried: